index.js

```diff
@@ -22,20 +22,27 @@ function relPath(base, filePath) {
 }
 
 function getManifestFile(opts, cb) {
-	file.read(opts.path, opts, function (err, manifest) {
-		if (err) {
-			// not found
-			if (err.code === 'ENOENT') {
-				cb(null, new gutil.File(opts));
-			} else {
-				cb(err);
-			}
+	var f = new gutil.File(opts)
+	if(opts.merge) {
+		var p = typeof opts.merge == 'string'? opts.merge : opts.path
```

Comment:
How 'bout 

```js
var p = (typeof opts.merge == 'string') ? opts.merge : opts.path;
```

?

There are too many spaces on the left half of that ternary operator for my liking :)